JiraReporter/Services/DayLogLoader.cs

```diff
@@ -71,7 +71,7 @@ private void AddIssueToDaylog(JiraDayLog dayLog, IssueDetailed issue)
             issue.CopyTo<IssueDetailed>(currentIssue);
             currentIssue.ExistsInTimesheet = true;
 
-            currentIssue.Entries = issue.Entries.Where(e => e.AuthorFullName == _author.Name && e.StartDate.ToOriginalTimeZone(_context.OffsetFromUtc).Date >= _date && e.StartDate.ToOriginalTimeZone(_context.OffsetFromUtc) < _date.AddDays(1)).ToList(); 
+            currentIssue.Entries = issue.Entries.Where(e => e.AuthorFullName == _author.Name && e.StartedAt.ToOriginalTimeZone(_context.OffsetFromUtc).Date >= _date && e.StartedAt.ToOriginalTimeZone(_context.OffsetFromUtc) < _date.AddDays(1)).ToList(); 
```

Comment:
let's also separate this where into multiple queries based on && (same as we do for the RemoveAll stuff). foreach && we add a new .Where query instead.